core/image/common.odin

```diff
@@ -1438,6 +1441,17 @@ expand_grayscale :: proc(img: ^Image, allocator := context.allocator) -> (ok: bo
 	return true
 }
 
+vertical_flip :: proc(img: ^Image) {
+    pixels := img.pixels.buf[:]
+    bpp := img.depth/8 * img.channels
```

Comment:
This proc should probably have a comment that it only supports 8 and 16 bpp, and an assert at the top that enforces this.